com.unity.netcode.gameobjects/Tests/Runtime/IntegrationTestExamples.cs

```diff
@@ -0,0 +1,192 @@
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using Unity.Netcode.TestHelpers.Runtime;
+
+namespace Unity.Netcode.RuntimeTests
+{
+    public class IntegrationTestUpdated : NetcodeIntegrationTest
+    {
+        private GameObject m_MyNetworkPrefab;
+        protected override int NbClients => 1;
```

Comment:
can we just rename `NbClients` to `NumberOfClients` please?

Reply:
Yeah, we can do that.